content/ga360-features.md

```diff
@@ -54,44 +50,35 @@ version with much lowered cost and following benefits
-            <td class="au-table__cell">Raw Data</td>
-            <td class="au-table__cell au-table__cell">Not available</td>
-            <td class="au-table__cell au-table__cell">Ability to export through Big Query</td>
-        </tr>
-        <tr class="au-table__row">
-            <td class="au-table__cell">Digital service standard</td>
-            <td class="au-table__cell au-table__cell">Not defined</td>
-            <td class="au-table__cell au-table__cell">Defined</td>
+            <td class="au-table__cell au-table__cell">Not guaranteed</td>
+            <td class="au-table__cell au-table__cell">Guaranteed 4 hours (usually faster)</td>
         </tr>
         <tr class="au-table__row">
-            <td class="au-table__cell">Configuration</td>
-            <td class="au-table__cell au-table__cell">May require changes</td>
-            <td class="au-table__cell au-table__cell">No change</td>
+            <td class="au-table__cell">Sampling Thresholds</td>
+            <td class="au-table__cell au-table__cell">500k sessions per date range</td>
+            <td class="au-table__cell au-table__cell">100M sessions per date range</td>
         </tr>
         <tr class="au-table__row">
             <td class="au-table__cell">Privacy</td>
-            <td class="au-table__cell au-table__cell">Maintained with protocols according to law</td>
-            <td class="au-table__cell au-table__cell">Maintained with protocols according to law</td>
+            <td class="au-table__cell au-table__cell">Basic</td>
+            <td class="au-table__cell au-table__cell">Enhanced and stricter privacy features</td>
         </tr>
         </tbody>
     </table>
+    <p>* Comparison data gathered from the <a href="https://marketingplatform.google.com/about/analytics/compare/" target="_blank">Google Marketing Platform</a> and the <a href="https://www.dta.gov.au/our-projects/google-analytics-government" target="_blank">DTA Google Analytics page</a>.</p>
 </div>
 <br/>
 
 ## Pricing Model
 
-Observatory has a whole-of-government contract, each new agency that joins
-unlocks more savings for everyone subscribed. The free tier of service we offer
-means that smaller agencies can save time and money. Note: Google makes Google
-Analytics free of charge up to a maximum of 10 million Hits per month per
-account.
+Observatory has a whole-of-government contract that leverages economies of scale
+to deliver better pricing for subscribers. The free tier of service that we
+offer helps smaller agencies save time and money. _Note: Google Analytics is
+free of charge up to a maximum of 10 million hits per month per account._
 
-Our contract fees are calculated on a cost recovery basis, based on the number
-of billable hits per month across all of a subscribed agencies websites.
+Our contract fees are calculated on a cost-recovery basis, based on the number
+of billable hits per month across all of the websites owned by a subscribed
+agency.
```

Comment:
billable hits are only calculated on the properties that are upgraded to GA360.

e.g. An agency can have 100 properties, but only decides to updrage 1 of them. They are only charged for the property they've upgraded to GA360

Reply:
Changed the above to "... across all of the properties upgraded to Google Analytics 360 by a subscribed agency." 
Is that sufficient?